sdks/python/apache_beam/runners/common.py

```diff
@@ -665,7 +663,7 @@ def process_outputs(self, windowed_input_element, results):
     if results is None:
       # TODO(BEAM-3937): Remove if block after output counter released.
       # Only enable per_element_output_counter when counter cythonized.
-      if (self.per_element_output_counter and
+      if (self.per_element_output_counter is not None and
```

Comment:
Hey @robertwb, could you please explain a little bit here why this caused perf changes? Thanks so much for your help!

Reply:
Sure. 

Writing `if expression:` (or using it in a logical construction with ands/ors) causes `bool(expression_result)` to be called. Generally this is just fine and preferred, but given the frequency this is called we can instead check if it's (not) exactly `None` which compiles down to a single pointer comparison. 

I'll merge this as soon as I verify test (still) pass.

Reply:
Thanks so much for your explaination! We are still discussing about how to use experimental flag. We are supposed to reach to an agreement and update this PR for the experimental flag part before this Thursday.